hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/AccountInfoMigration.java

```diff
@@ -0,0 +1,190 @@
+package com.hedera.mirror.importer.migration;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.google.common.base.Stopwatch;
+import com.hederahashgraph.api.proto.java.CryptoGetInfoResponse.AccountInfo;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.zip.GZIPInputStream;
+import javax.inject.Named;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.lang3.StringUtils;
+import org.flywaydb.core.api.MigrationVersion;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
+
+import com.hedera.mirror.importer.MirrorProperties;
+import com.hedera.mirror.importer.domain.Entities;
+import com.hedera.mirror.importer.domain.EntityId;
+import com.hedera.mirror.importer.domain.TransactionTypeEnum;
+import com.hedera.mirror.importer.repository.EntityRepository;
+import com.hedera.mirror.importer.repository.TransactionRepository;
+import com.hedera.mirror.importer.util.Utility;
+
+@Named
+@RequiredArgsConstructor(onConstructor_ = {@Lazy})
+public class AccountInfoMigration extends MirrorBaseJavaMigration {
+
+    private static final Collection<Integer> TRANSACTION_TYPES = Set.of(
+            TransactionTypeEnum.CONTRACTCREATEINSTANCE.getProtoId(),
+            TransactionTypeEnum.CRYPTOCREATEACCOUNT.getProtoId(),
+            TransactionTypeEnum.FILECREATE.getProtoId()
+    );
+
+    @Value("classpath:accountInfo")
+    private Path accountInfoDirectory;
+
+    private final EntityRepository entityRepository;
+    private final MirrorProperties mirrorProperties;
+    private final TransactionRepository transactionRepository;
+
+    @Override
+    public Integer getChecksum() {
+        return 1; // Change this if this migration should be re-ran
```

Comment:
nit, [had to look this up to be sure](https://english.stackexchange.com/a/251447)
```suggestion
        return 1; // Change this if this migration should be rerun
```

Reply:
Updated

Reply:
Did you push?  Still seeing re-ran.

Reply:
No push

Reply:
Pushed